pkg/tanka/tanka.go

```diff
@@ -27,8 +27,13 @@ type options struct {
 
 	// additional options for diff
 	diff kubernetes.DiffOpts
+
 	// additional options for apply
 	apply kubernetes.ApplyOpts
+
+	// additional options for delete
+	// this is type-aliased to kubernetes.ApplyOpts and will be removed.
+	delete kubernetes.DeleteOpts
```

Comment:
You can remove this field altogether. AFAICT it's not used by any code and is totally not required.

Reply:
Fixed.